src/routes/safe/store/actions/transactions/utils/newTransactionsHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  EthereumTransaction,
+  ModuleTransaction,
+  MultiSigTransaction,
+  Transaction,
+  TxType,
+} from '../../../models/types/transactions'
+
+export const isMultiSigTx = (tx: Transaction): tx is MultiSigTransaction => {
```

Comment:
is keyword doesn't seem to do what you want
https://stackoverflow.com/questions/40081332/what-does-the-is-keyword-do-in-typescript

Reply:
good catch thanks!